fastchat/llm_judge/common.py

```diff
@@ -484,6 +491,42 @@ def chat_compeletion_anthropic(model, conv, temperature, max_tokens):
     return output.strip()
 
 
+def chat_completion_cohere(model, conv, temperature, max_tokens):
```

Comment:
why add it here but not in `api_provider.py`?
https://github.com/lm-sys/FastChat/blob/main/fastchat/serve/api_provider.py

Reply:
Good call out. I did not require streaming support for my experiments. Maybe we can leave this as a future TODO?